### DISCUSSION TITLE
Help🙏 Build error occurred when I try to dep

### DISCUSSION THREAD

**Discussion by Mao718** [morethanmin/morethan-log#99] [Q&A] [answered]
Here's the error look like.
![image](https://user-images.githubusercontent.com/60081713/217999487-dfcdfbd6-47e4-402e-8377-c33c7d35f039.png)

And here's the config I have changed
![image](https://user-images.githubusercontent.com/60081713/217999610-cad448cb-3c0f-49b9-8735-75d29c7f6247.png)

Hope I didn't massy up in changing the config

**Comment by Mao718** [comment, marked as answer]
The problem are solved.
I miss fill the NOTION_PAGE_ID.

**Reply by morethanmin** [reply]
nice! 👍
